style.css

```diff
@@ -189,3 +189,92 @@ section .heading-2 {
 #two {
   padding-left: 202px;
 }
+
+.formsection {
+  background-image: url('images/Contact\ me.jpg');
+  width: 300px;
+  margin-top: 300px;
+  background-repeat: no-repeat;
+  height: 779px;
+}
+
+.formsection h1 {
+  padding-top: 100px;
+  padding-left: 43px;
+  width: 290px;
+  height: 56px;
+  left: -213px;
+  top: 4734px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.formsection p {
+  width: 290px;
+  height: 102px;
+  left: 24px;
+  top: 4802px;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  display: flex;
+  padding-left: 5px;
+  padding-top: 10px;
+}
+
+.contact-box {
+  width: 100%;
+  grid-template-columns: repeat(2, 1fr);
+  padding-top: 80px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- According to the project requirements, this section of the page should be aligned using the `flex` display property, this would allow for more responsiveness. Please fix this 👍 👍